README.md

```diff
@@ -1,3 +1,5 @@
+![Travis Build Status](https://img.shields.io/travis/com/open-quantum-safe/liboqs?logo=travis&label=travis%20ci)
```

Comment:
Yes this would nice. I think each workflow would need their own badge and we have quite a few (I count 18). Any thoughts if we want badges for all of them, or only a subset?
For example: android, apple, macos, platforms, weekly, windows, zephyr, downstream-basic, downstream-release.

Reply:
I would suggest the [main branch tests](https://github.com/open-quantum-safe/liboqs/actions/workflows/commit-to-main.yml) and the [weekly tests](https://github.com/open-quantum-safe/liboqs/actions/workflows/weekly.yml). These include all of the platform tests and the constant-time / extended KAT tests.

Reply:
Opened #1984, and in the meanwhile removed the Travis badge to avoid the impression that there is only one CI workflow in liboqs.

Reply:
So would it be sensible to add Travis badge again now that the GH CI badges returned?